library/coretests/tests/ptr.rs

```diff
@@ -1049,38 +1049,37 @@ fn test_ptr_default() {
 #[test]
 fn test_const_drop_in_place() {
```

Comment:
This test doesn't fit the ptr module anymore. I like the new test, and don't think the old test is worth keeping, so please move it either to somewhere we have manuallydrop tests or where we have cell tests